styles/ligth_dark.css

```
@@ -0,0 +1,56 @@
+.icon-moon {
+    width: 30px;
+    cursor: pointer;
+    margin-right: 10px;
+}
+
+.btn_toggle {
+    display: flex;
+    align-items: flex-end;
+    justify-content: center;
+}
+
+.light_dark.dark {
+    background-color: aliceblue;
+}
+
+body.dark {
+    background-color:  #191919;
+    transition: 0.5s;
+}
+
+body.dark .navigation,
+.nav-mobile-icons {
+    background: #191919;
+}
+
+body.dark .content,
+body.dark .tech-stack {
+    color: #cccccc;
+}
+
+
+body.dark .icon-box .icon-github {
+    background-image: url('../images/akar-icons_github-fill\ \(88x88\)\ tech\ dark.svg');
```

é melhor renomear esse svg pra remover caracteres especiais
```suggestion
    background-image: url('../images/github_dark_icon.svg');
```

---

Esse trecho era pra ter sido retirado, eu troquei a logo e coloquei uma que se aplica nos dois modos...